### ISSUE TITLE
Incorrect authentication error message

### ISSUE THREAD

**Issue by minhdtb** [wundergraph/wundergraph#39]
### Bug description

incorrect authentication error when RBAC is not sufficient

### How to reproduce

Create a simple WunderGraph application from template and use open id connect auth provider

Operation

```graphql
query @rbac(requireMatchAll: [tester,admin]) {
    findUsers: db_findManyuser {
        id
        email
        name
    }
}
```
create a hook in wundergraph.server.ts
```ts
export default configureWunderGraphServer<HooksConfig, InternalClient>((serverContext) => ({
    hooks: {
        authentication: {
            mutatingPostAuthentication: async (user) => {
                return {
                    status: "ok",
                    user: {
                        ...user,
                        roles: [
                            "tester",
                        ]
                    }
                }
            }
        },
        queries: {},
        mutations: {},
    },
    graphqlServers: [],
}));
```
index.tsx
```ts
import { NextPage } from 'next';
import styles from '../styles/Home.module.css';
import { AuthProviders, useQuery, useWunderGraph, withWunderGraph } from '../components/generated/nextjs';

const Home: NextPage = () => {
    const {result: test} = useQuery.Test();
    const {user, login, logout} = useWunderGraph();
    return (
        <div className={styles.container}>
            <button onClick={() => login(AuthProviders.storeo)}>Login</button>
            <button onClick={() => logout({
                logout_openid_connect_provider: true
            })}>Logout
            </button>
            <h1 className={styles.title}>Hello WunderGraph! {user?.user_id}</h1>
            {test.status === "ok" && test.data.findUsers.length !== 0 && (
                <div>
                    {test.data.findUsers.map(message => {
                        return (<div key={message.id}>
                            <p>
                                id: {message.id}, email: {message.email}, name: {message.name}
                            </p>
                        </div>)
                    })}
                </div>
            )}
            {test.status === "error" && (
                <div>
                    {test.errors.map(value => {
                        return (<div key={value.message}>
                            <p>
                               message: {value.message}, path: {value.path}
                            </p>

                        </div>)
                    })}
                </div>
            )}
        </div>
    );
};

export default withWunderGraph(Home);
```
Actual error message: 
```
Unexpected token u in JSON at position 0
```

### Expected behavior

The error message should be ```Access denied``` or something related to exact problem

### Environment & setup

- OS: Ubuntu
- Go version: go1.17.2 linux/amd64
- Database: PostgreSQL
- Node.js version: v16.15.1

### WunderCtl Version

```
Version: 0.90.32
Commit: 386a3d4b66204411dc89da6919aad0bebe524657
Date: 2022-06-06T12:21:43Z
BuiltBy: ci

```

**Comment by jensneuse** [comment]
Your observation is correct, this needs fixing. Putting it on the backlog.

**Comment by jensneuse** [comment]
Fixed with @wundergraph/nextjs@0.0.28
